{{cookiecutter.project_slug}}/backend/app/api/api_v1/routers/tests/test_auth.py

```diff
@@ -14,6 +14,21 @@ def verify_password_mock(first: str, second: str):
     assert response.status_code == 200
 
 
+def test_signup(client):
```

Comment:
can you monkeypatch the hashing function here like I did in test_login (or verify_password)? it adds about a second or so per test and we don't need to test that it hashes correctly?

Reply:
The already exists test can also use the test_user fixture and be a separate test and I don't think it should add much time.